### PR TITLE
postgres: Adding docs for creating a read only user.

### PR DESCRIPTION
**Pull request by richard-joerger-aiven**
# What changed, and why it matters
Added a page on how to create read only users' on our platform. Helpful for deflecting support requests. Reached out to the SME's about some possible approaches for creating read only users on our platform. Useful in cases where a customer wants to limit a users' ability to access data.

### CODE REVIEW

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]
Many thanks for adding this instructions :-)

File: docs/products/postgresql/howto/readonly-user.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Read Only User for PostgreSQL
+=============================
+In the interest of having users with the least permissions to complete their tasks, one may need a user with read only access to the whole database or a handful of tables. In some cases, we may want this to happen automatically, below are two approaches to complete this task.
+
+All new objects shall have a role with read-only permissions
+------------------------------------------------------------
+1. Alter the default permissions for the role for the given schema: ``ALTER DEFAULT PRIVILEGES FOR ROLE <target role> IN SCHEMA <schema name> abbreviated_grant_or_revoke``
```
[the remark this answers is not in the document]
Comment (richard-joerger-aiven) [reply docs/products/postgresql/howto/readonly-user.rst:7]: RE: abbreviated_grant_or_revoke, yes that is something the user would have to replace with their desired permissions. Shall I replace it with `<grant or revoke>`?

File: docs/products/postgresql/howto/readonly-user.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+2.  To update any existing database objects, run the following: ``GRANT SELECT ON ALL TABLES IN SCHEMA <schema name> to <myreadonlyrole>;``
+
+Only certain databases should be read-only for users in a particular role:
+==========================================================================
+1. Create a new database which will be used as a template ``CREATE DATABASE ro_<name>_template...``
```
[the remark this answers is not in the document]
Comment (richard-joerger-aiven) [reply docs/products/postgresql/howto/readonly-user.rst:13]: The standard information in this case would be the specific permissions that the CU would like to have be part of their template. Shall I say apply standard permissions & roles instead?

[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/postgresql/howto/readonly-user.rst:23]: Thanks, that makes sense! how about
`2. For the new template database, set permissions and roles that you want as default ones in the template.`

File: docs/products/postgresql/howto/readonly-user.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/postgresql/howto/readonly-user.rst]: Let's use
`YOUR_GRANT_OR_REVOKE_PERMISSIONS`

[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/postgresql/howto/readonly-user.rst]: Also, usually we use all-caps (a word with all the letters in the upper case) for variables or placeholders (as YOUR_GRANT_OR_REVOKE_PERMISSIONS). I see you use it for whole queries. Is that possible for you to update the queries and remove all-caps words/expressions unless required by the SQL standard?

File: docs/products/postgresql/howto/list-dba-tasks.rst [hunk 3]
```diff
@@ -39,7 +39,7 @@ Database administration tasks
         :shadow: md
         :margin: 2 2 0 0
 
-    .. grid-item-card:: :doc:`Use the PostgreSQL® pg_repack extension </docs/products/postgresql/howto/use-pg-repack-extension>`
+    .. grid-item-card:: :doc:`Use the PostgreSQL® pg_repack extension </docs/productspostgresql/howto/use-pg-repack-extension>`
```
Comment (wojcik-dorota) [inline docs/products/postgresql/howto/list-dba-tasks.rst:42]: In `</docs/productspostgresql/howto/use-pg-repack-extension>`, the slash is missing after `products`.

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]

**Review by wojcik-dorota — APPROVED** [review]
Thank you :-)

**Pull request merged**